tests/src/invalid/constantUsages.php

```diff
@@ -15,4 +15,4 @@
 
 /** @var class-string<DateTimeZone> $tz */
 $tz = DateTimeZone::class;
-$tz::UTC;
+$tz::FTC;
```

Comment:
ℹ️ I've opted for still reporting about error if trying to access const that is not defined on given class

Reply:
Cool, like the constant name 😂